clipping_manager/templates/clipping_manager/browse.html

```diff
@@ -1,5 +1,6 @@
 {% extends "sidebar.html" %}
-{% load cms_tags i18n crispy_forms_tags %}
+{% load cms_tags i18n crispy_forms_tags%}
+{% load static %}
```

Comment:
It might be that this doesn't resolve to working static files. Not sure though. 

`staticfiles` supports non-local file backends and I think the divio production system uses some AWS backend for the statics.

Reply:
I believe _staticfiles_ is depracated for newer versions of Django, but you're using an older version here, so you might be right

Reply:
Might be! I'm not actively working with the Django versions of the last years sadly :cry: